docs/container.md

```diff
@@ -63,12 +64,12 @@ class Config extends ContainerConfig
     {
         $adr = $di->get('radar/adr:adr');
 
+        $adr->middle('Relay\Middleware\ResponseSender');
```

Comment:
So does that mean, it fails to have the `Relay\Middleware\ResponseSender` being the first middleware and `Execution` being second one?